k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -452,7 +452,7 @@ public static void save(StorageEditor editor) {
         editor.putBoolean("useVolumeKeysForListNavigation", useVolumeKeysForListNavigation);
         editor.putBoolean("autofitWidth", autofitWidth);
         editor.putBoolean("quietTimeEnabled", quietTimeEnabled);
-        editor.putBoolean("notificationDuringQuietTimeEnabled", notificationDuringQuietTimeEnabled);
+        editor.putBoolean("notificationDuringQuietTimeDisabled", notificationDuringQuietTimeDisabled);
```

Comment:
You can't just rename that setting's key. Existing installations use the old name. Their setting will reset to the default value if you change the name.

Reply:
Ok, I will then change only the variable name and keep the old name for the key.
The current name of the variable causes a misunderstanding of the code, because when the variable is true the notifications should be disabled.